tests/Aspire.Hosting.Tests/DistributedApplicationTests.cs

```diff
@@ -315,11 +315,12 @@ public async Task VerifyDockerAppWorks()
     [ActiveIssue("https://github.com/dotnet/aspire/issues/4651", typeof(PlatformDetection), nameof(PlatformDetection.IsRunningOnCI))]
     public async Task VerifyContainerStopStartWorks()
     {
-        using var testProgram = CreateTestProgram(randomizePorts: false);
+        using var testProgram = CreateTestProgram("container-start-stop", randomizePorts: false);
 
         testProgram.AppBuilder.Services.AddLogging(b => b.AddXunit(_testOutputHelper));
 
-        testProgram.AppBuilder.AddContainer("redis0", "redis")
+        const string containerName = "container-start-stop-redis";
```

Comment:
Do container names need to be unique? That doesn't seem nessessary if DCP already appends a unique ID onto each name.

Reply:
No they do not need to be strictly unique. The point of the change is that if a container is left running after running a bunch of tests, I want to know which test leaked the container by just looking at its name.